CHANGES.md

```diff
@@ -10,6 +10,13 @@ creating a new release entry be sure to copy & paste the span tag with the
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
 
+
+## __cylc-8.2.1 (<span actions:bind='release-date'>Pending</span>)__
+
+[#5571](https://github.com/cylc/cylc-flow/pull/5571) - Make workflow `CYLC_`
+variables available to the template processor during parsing.
+
+
```

Comment:
Unless any objections to #5634 then I suggest the description should instead be written to `changes.d/5571.feat.md`.

Reply:
No objections. I've just merged that PR.  I'll draft this briefly until I've added a unit test and rejigged the change log.

Reply:
Done.